recipes-go/3mdeb-rtectrl/3mdeb-rtectrl_git.bbappend

```diff
@@ -3,11 +3,13 @@ FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
 SRC_URI += " \
     file://RteCtrl.cfg \
     file://RteCtrl.service \
+    file://rte_ctrl.sh \
```

Comment:
This is wrong, if the script is in the RteCtrl repo, we should use it from there. This line should be removed.

Reply:
rte_ctrl.sh removed from meta layer, I will resolve this conversation